From 9cb588698759b541c3dd6796131f7dc557f8798b Mon Sep 17 00:00:00 2001
From: Don Zickus <dzickus@redhat.com>
Date: Tue, 16 Sep 2014 17:04:28 -0400
Subject: [usb] ehci: no conditional compilation for interestingness

Message-id: <1410887463-20674-197-git-send-email-dzickus@redhat.com>
Patchwork-id: 95408
O-Subject: [RHEL7 PATCH 196/591] ehci: no conditional compilation for interestingness
Bugzilla: 1110939
RH-Acked-by: Tony Camuso <tcamuso@redhat.com>
RH-Acked-by: Prarit Bhargava <prarit@redhat.com>

https://bugzilla.redhat.com/show_bug.cgi?id=1110939

commit bbcd5cab03e61f69d50a1535ff2646f9698b9760
Author: Oliver Neukum <oneukum@suse.de>
Date:   Mon Nov 18 13:23:01 2013 +0100

    ehci: no conditional compilation for interestingness

    Simple elemination of the conditional compilation

    Signed-off-by: Oliver Neukum <oneukum@suse.de>
    Signed-off-by: Greg Kroah-Hartman <gregkh@linuxfoundation.org>
---

Signed-off-by: Jarod Wilson <jarod@redhat.com>

diff --git a/drivers/usb/host/ehci-hub.c b/drivers/usb/host/ehci-hub.c
index 3dcc0ab..d594db3 100644
--- a/drivers/usb/host/ehci-hub.c
+++ b/drivers/usb/host/ehci-hub.c
@@ -1114,10 +1114,8 @@ static int ehci_hub_control (
   if (test_bit(wIndex, &ehci->port_c_suspend))
    status |= USB_PORT_STAT_C_SUSPEND << 16;
 
-#ifndef VERBOSE_DEBUG
- if (status & ~0xffff) /* only if wPortChange is interesting */
-#endif
-  dbg_port (ehci, "GetStatus", wIndex + 1, temp);
+  if (status & ~0xffff) /* only if wPortChange is interesting */
+   dbg_port(ehci, "GetStatus", wIndex + 1, temp);
   put_unaligned_le32(status, buf);
   break;
  case SetHubFeature:
-- 
1.7.1